docker-compose.yml

```
@@ -8,15 +8,24 @@ services:
     init: true
 
     restart: always
+    depends_on:
+      - redis
 
     environment:
-      - SEASONALBOT_TOKEN=yourtokenhere
-      - SEASONALBOT_DEBUG=true
-    # - SEASONALBOT_GUILD=
-    # - SEASONALBOT_ADMIN_ROLE_ID=
-    # - CHANNEL_ANNOUNCEMENTS=
-    # - CHANNEL_DEVLOG=
+      - SEASONALBOT_TOKEN=${SEASONALBOT_TOKEN}
+      - SEASONALBOT_DEBUG=${SEASONALBOT_DEBUG}
+      - SEASONALBOT_GUILD=${SEASONALBOT_GUILD}
+      - SEASONALBOT_ADMIN_ROLE_ID=${SEASONALBOT_ADMIN_ROLE_ID}
+      - CHANNEL_DEVLOG=${CHANNEL_DEVLOG}
+      - CHANNEL_SEASONALBOT_COMMANDS=${CHANNEL_SEASONALBOT_COMMANDS}
```

You can just specify the environment variable itself if you just want it to be carried over from the host:
```suggestion
      - SEASONALBOT_TOKEN
      - SEASONALBOT_DEBUG
      - SEASONALBOT_GUILD
      - SEASONALBOT_ADMIN_ROLE_ID
      - CHANNEL_DEVLOG
      - CHANNEL_SEASONALBOT_COMMANDS
```

---

Relevant docs for this feature: https://docs.docker.com/compose/environment-variables/#pass-environment-variables-to-containers